posts/2022-01-20-beta-release-post.adoc

```diff
@@ -0,0 +1,100 @@
+seo-description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+blog_description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+[#allbeta]
+== All Beta Features package
+The All Beta features package for link:{url-about}[Open Liberty] contains all Open Liberty beta features and GA features and functions.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try the beta package, <<feedback, let us know what you think>>.
+
+The following beta features are available in 22.0.0.2-beta:
+
+* <<ssl, Support new change in the SSL certificate with urlencoded string>>
+
+[#ssl]
+=== Support new change in the SSL certificate with urlencoded string
+
+An `X.509` certificate is a digital certificate which is used to manage identity and security in internet communications and computer networking.
+
+The `X.509` certificate can be sent by the intermediate servers (like web server, proxy server or load balancer) to an application server. Due to the deprecation of long lines folding in the `HTTP` header continuation, some of the open source intermediate servers start sending a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate to application server.
+
+Currently, WebSphere Application Server and Liberty Server do not support this new url-encoded certificate. Any request with this new format is rejected and the request is terminated.
+
+An example of a truncated url-encoded certificate sent to application server:app-name:
+
+[source]
+----
+[-----BEGIN%20CERTIFICATE-----%0AMIIDODCCAiCgAwIBAgIIHbO76YfCH1cwDQYJKoZIhvcNAQELBQAw
+OjELMAkGA1UE%0ABhMCVVMxDDAKBgNVBAoTA0lCTTEMMAoGA1UECxMDVFdTMQ8wDQYDVQQDEwZDbGll%0AbnQ
+wHhcNMTUxMTAzMTYyODM1WhcNMzUxMDMwMTYyODM1WjA6MQswCQYDVQQGEwJV%0AUzEMMAoGA1UEChMDSUJNM
+QwwCgYDVQQLEwNUV1MxDz%2F%2Bpo%2FF%2Bzi%2F0sfAUwRdfgMm%2FAKAGzwMQufOYeKCgMULtq14QAJQnL
+mq4M%2FM00%0AC5QyeYtdaTou%2BMsLmoa1tkq2VSDVxAcktJyRSRsox36G7EHDLV4U2gtR6xczNEjw%0A2%2
+Bsj772FjdAMXRSR%0A-----END%20CERTIFICATE-----%0A]
+----
+
+Note: 
+
+* (1) The header (i.e BEGIN ) and footer (i.e END) information are appended before sending to application server.
+* (2) The encoded character (i.e %) presents in the certificate.
+
+The IBM application server code is updated to auto detect and process this new format.
```

Comment:
weird to see a mention of "IBM application server code"- shouldn't it just say Open Liberty? Need to confirm with dev